apps/widget-e2e/src/util/ScatterChartErrorBars.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Chart, IChartElement } from "./Chart";
+
+const dReg =
+  /^M([\d.]+),([\d.]+)h([\d.]+)m-([\d.]+),([\d.]+)V([\d.]+)m-([\d.]+),([\d.]+)h([\d.]+)$/;
+
+export interface IScatter extends IChartElement {
+  readonly x?: number;
+  readonly y1?: number;
+  readonly y2?: number;
+  readonly y3?: number;
+}
+
+export class ScatterChartErrorBars extends Chart<IScatter> {
+  public get Elements(): IScatter[] {
+    return this.getHtmlElements(".trace.scatter:eq(0) .errorbars path").map(
+      (b, i) => this.getCoordinate(b, i)
+    );
+  }
+
+  public sortByH(): IScatter[] {
+    return this.Elements.sort((a, b) => a.top - b.top);
+  }
+  public clickNthPoint(idx: number): void {
+    const offset = this.getNthPointOffset(idx);
+    if (!offset) {
+      return;
+    }
+    cy.get(`${this.container} .nsewdrag.drag`).trigger("mouseover", {
```

Comment:
wow, this is hovering on points in the scatter plot to trigger mouseover events? very fancy test code

Reply:
yeas, mouseover along with trigger created hover effect. But I am yet to find the correct offset in this case. It is not completely producing hover result yet. I will have tweak a little bit to provide hover result.